### ISSUE TITLE
Analyze the need of wedeploy-runc fork

### ISSUE THREAD

**Issue by eduardolundgren** [wedeploy/wedeploy-runc#1]

**Comment by cirocosta** [comment]
Hey,

our fork of `runc` is still needed given that Docker didn't implement some options yet (see https://github.com/moby/moby/issues/25303). 

Although the patch is reasonably simple (https://github.com/cirocosta/runc/commit/3f021d68a18d6cd1049ae74e35ab4feb37dcb4e7), it does a great job of protecting the host from fork bombs. 

In the first commit I had also set memory swap to `0` (such that we don't stick with the default docker behavior or giving one extra `N` amount of swap to the container by default - where `N` is the amount of regular memory it has), but that had lead hosts to continually kill containers (as many Liferay ones would get to the maximum memory when initializing).

I just rebased the code and published a GitHub release on my fork (https://github.com/cirocosta/runc/releases/tag/wedeploy-0.1.1). 

I'm updating `wedeploy/training` with information pertaining how the hosts are configured, so `wedeploy-runc` will be covered there :+1:
